rest/taskana-rest-spring-example-common/pom.xml

```diff
@@ -14,13 +14,28 @@
 		<version>1.1.6-SNAPSHOT</version>
 		<relativePath>../pom.xml</relativePath>
 	</parent>
-
+	<dependencyManagement>
```

Comment:
Why did you add this? Personally I do not like using dependencyManagement unless it is necessary

Reply:
lets postpone this to another ticket. tried to get rid of it.

Reply:
please create it :)